README.md

```diff
@@ -143,6 +143,23 @@ You'll also need an alias `kojikat` to point to:
    targeted platforms and required SCL + non-SCL modes.
 1. Submit a pull request against `rpm/develop`
 
+## HOWTO: build multiple packages
+
+If you have multiple packages that you want to build together because they
+depend on each other, `mockchain` can be very helpful. To do this:
+
+1. Use tito to generate a SRPM for every package. Run  `tito build --srpm
+--test --builder tito.builder.MockBuilder --arg mock_config_dir=mock/ --arg
```

Comment:
I don't think any of the --builder or mock arguments are required, using `tito build --srpm --test` is sufficient.